timify-api/updatePassword.js

```diff
@@ -0,0 +1,17 @@
+import bcrypt from 'bcrypt';
+import {User }from './models/user.js';
+
+async function migrateUsers() {
+  try {
+    const users = await User.findAll(); 
+    for (const user of users) {
+      const hashedPassword = await bcrypt.hash(user.password, 10);
```

Comment:
What is 10 for in this function? just curious! is it the salt?

Reply:
and could we potentially wrap it in a variable with a name that explains its purpose?

Reply:
> What is 10 for in this function? just curious! is it the salt?

yes it's the salt